test/target.spec.js

```diff
@@ -36,6 +36,12 @@ module.exports.addTests = function({testRunner, expect}) {
       expect(allPages).toContain(page);
       expect(allPages[0]).not.toBe(allPages[1]);
     });
+    it('should allow background_page target type to pass through', async({browser}) => {
+      const targets = browser.targets();
+      targets[0]._targetInfo.type = 'background_page';
```

Comment:
let's be real: can you please create a minimalistic chrome extension under `/test/assets/simple-extension` and load it in the headful browser?

Reply:
Yes, will do it later today

Reply:
I have updated the test to use a headfull browser with a bare extension. I'm not sure about starting a new browser instance in a spec that already accepts a headless browser instance, Should I leave the test where it is or move it to another spec?